### PR TITLE
Add legal IPTV Links for testing

### PR DESCRIPTION
**Pull request by Artiume**

### CODE REVIEW

**Comment by Artiume** [comment]
Just tried to connect some of these to JF and JF wouldn't accept the URLs. Not sure what's up.

**Comment by PrplHaz4** [comment]
The first ABC news channel is my reference channel and it has always worked for me. I imagine they go stale depending on the provider. Pretty sure vlc can be used to check if it's a Jellyfin issue or link issue.

**Comment by Artiume** [comment]
Can I get the exact url? It wasn't a dead channel issue, it was an invalid url issue

**Comment by PrplHaz4** [comment]
Think this is it. https://abclive1-lh.akamaihd.net/i/abc_live01@423395/master.m3u8

I think live.m3u8 might be a better list as the intention is to test live streams.

**Comment by Artiume** [comment]
Updated to the live url

So i had to remove https and make it http for it to accept it as a _valid_ url. So docker is broken af. It might be that ssl issue, im not sure. Using the same method of making the url http did not allow me to add any other of the lists directly in JF, just the url you provided me. Testing the links on my ubuntu native install, i had zero issues with https. 

```
[2020-01-12 23:07:55.310 -05:00] [ERR] Error processing request
System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception.
 ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.
   at System.Net.Security.SslStream.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslStream.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.StartReceiveBlob(Byte[] buffer, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.StartReceiveBlob(Byte[] buffer, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslStream.EndProcessAuthentication(IAsyncResult result)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Http.ConnectHelper.EstablishSslConnectionAsyncCore(Stream stream, SslClientAuthenticationOptions sslOptions, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.EstablishSslConnectionAsyncCore(Stream stream, SslClientAuthenticationOptions sslOptions, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsyncInternal(HttpRequestOptions options, HttpMethod httpMethod)
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsync(HttpRequestOptions options, HttpMethod httpMethod)
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.Get(HttpRequestOptions options)
   at Emby.Server.Implementations.LiveTv.TunerHosts.M3UTunerHost.Validate(TunerHostInfo info)
   at Emby.Server.Implementations.LiveTv.LiveTvManager.SaveTunerHost(TunerHostInfo info, Boolean dataSourceChanged)
   at MediaBrowser.Api.LiveTv.LiveTvService.Post(AddTunerHost request)
   at Emby.Server.Implementations.Services.ServiceExecGeneral.GetTaskResult(Task task)
   at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost httpHost, IRequest httpReq, HttpResponse httpRes, ILogger logger, CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken)
```

File: general/server/live-tv/setup-guide.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ This tuner allows you to add IPTV channel to Jellyfin by using the appropriate M
 
 * Auto-loop live streams is sometimes necessary for some IPTV channels.  Turn this on only if your streams are not playing correctly
 
+> [!NOTE]
+> Here is a github for legal samples to use to test connectivity
```
Comment (dkanada) [inline general/server/live-tv/setup-guide.md]:
```suggestion
> Here is a list of legal samples to use to test connectivity.
```

**Comment by JustAMan** [comment]
> It might be that ssl issue, im not sure

Can it be the problem https://github.com/jellyfin/jellyfin/pull/1960 tries to address?

**Comment by Artiume** [comment]
I had loaded it, spun up the container and appeared to have it still. But I didn't give it proper testing because that's when I noticed my corruption of my pr due to master being in my pr and focused on fixing that.

**Comment by Artiume** [comment]
I also really really want to not use docker because the ssl bug is messing with comms with metadata providers but I'm living with it to watch the bug's behavior. My logs are pretty impossible to parse because they're so flooded lol. If I want to review anything, I have to recreate an event rather than check the logs first. You guys have a good log program for live feed the logs? I've been janky using the browser windows. I'd like to make a live feed log in JF but that's not where my skills lay

**Pull request merged**